userspace/libsinsp/cgroup_limits.cpp

```diff
@@ -5,12 +5,19 @@
 #include "sinsp.h"
 
 namespace {
+
+#ifdef CONFIG_ARM
+ // @kris-nova if we are running on arm architecture let's dial the limit back substantially
+ // note: a smaller limit was required to build on RaspberryPi 4 armv7l
+constexpr const int64_t CGROUP_VAL_MAX = (1UL << 24u) - 1;  
```

Comment:
@kris-nova can you try
```
constexpr const int64_t CGROUP_VAL_MAX = (1ULL << 42u) - 1;
```

instead (make the 1 an explicit unsigned long long)? If it works for you on ARM, we can change this unconditionally instead of introducing CONFIG_ARM.

The issue is probably that `long` is 32 bits on ARM (and 64 bits on amd64).

Reply:
Good call - I will test and adjust